### PR TITLE
Feature/repository improvements

### PR DESCRIPTION
**Pull request by NoeliaBentancor**
## Description

- Delete manifest samples .

They are not referenced in code, so they should not be placed here.

- Place type mappers in types/mappers folder so as to achieve a better understanding of the code.

## Requirements Checklist
- [ ] Unit Tests updated or fixed
- [ ] Docs/guides updated

### CODE REVIEW

**Comment by mpollingerQualabs** [comment]
I would argue the sample manifest files should be somewhere in the repo, if they don't belong in the `src` directory that's ok, but maybe place them somewhere in a `test` directory, not to lose them.

**Comment by NoeliaBentancor** [comment]
> I would argue the sample manifest files should be somewhere in the repo, if they don't belong in the `src` directory that's ok, but maybe place them somewhere in a `test` directory, not to lose them.

The issue is that we are not using or refer them in any place (even when doing tests).
When doing tests, we have some files that have dash or hls object.

**Comment by mpollingerQualabs** [comment]
> > I would argue the sample manifest files should be somewhere in the repo, if they don't belong in the `src` directory that's ok, but maybe place them somewhere in a `test` directory, not to lose them.
> 
> The issue is that we are not using or refer them in any place (even when doing tests). When doing tests, we have some files that have dash or hls object.

As i commented on the PR that introduced those files for testing, what should be done is load the actual (manifest) files into the object and use those, instead of creating an object with the manifest as a string. Could be done with `fs` or any other tool that let's you load a file into an object. The current test files have the same contents as these files that were removed.

**Review by matiasrb97 — COMMENTED** [review]
@NoeliaBentancor let's keep just the changes related to repository organization and undo the deletion of the samples.

**Comment by NoeliaBentancor** [comment]
@matiasrb97 @mpollingerQualabs as we will be using them in tests, i readded them and then we can delete from src when merging testing pull request.

Let me know if it sounds good for you!

File: lib/src/cmaf/utils/hls/HLSToHam.ts [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+	SelectionSet,
+	SwitchingSet,
+	Track,
+	VideoTrack,
+} from '../../ham/types/model';
+import { addMetadataToHLS } from '../manifestUtils.js';
+import { PlayList } from '../../ham/types/mappers/HlsManifest.js';
+import { Manifest } from '../../ham/types/';
+
+function m3u8ToHam(manifest: Manifest) {
```
Comment (matiasrb97) [inline lib/src/cmaf/utils/hls/HLSToHam.ts]: I think we deleted this file because is not used. Can you confirm that?

Comment (NoeliaBentancor) [reply lib/src/cmaf/utils/hls/HLSToHam.ts]: That´s right. Thank you for noticing it!  I had it in my local branch and didn´t update. Just deleted.

**Pull request merged**